akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/ActiveActiveEventSourcing.scala

```diff
@@ -73,13 +68,14 @@ private[akka] class ActiveActiveContextImpl(val entityId: String, val replicaId:
    * Whether the happened concurrently with an event from another replica.
    * Undefined result if called from any where other than an event handler.
    */
-  override def concurrent: Boolean = _concurrent
+  override def concurrent: Boolean = throw new UnsupportedOperationException("TODO")
+
   override def persistenceId: PersistenceId = PersistenceId.replicatedUniqueId(entityId, replicaId)
+
   override def currentTimeMillis(): Long = {
-    // FIXME always increasing
-    System.currentTimeMillis()
+    WallClock.AlwaysIncreasingClock.currentTimeMillis()
```

Comment:
Now we added a single atomic that all entities on the same node will access, not sure if that matters, just noting given that is was just a local var before.

Reply:
Each could have their own as well, I'll leave it as is for now